### ISSUE TITLE
Quotes are now included as part of a word.  "w" treated as "W". 

### ISSUE THREAD

**Issue by MatthewCushing** [VSCodeVim/Vim#7427]
I'm not sure when exactly this happened, sometime in the past few weeks I'm sure.  I've never had this issue before but now every time I move forward by word, it will skip the beginning of the actual word and use the quotes surrounding the word as the beginning.  It also avoids punctuation like semicolons and treats them as part of a word.  Essentially, it is treating `w` as if it were `W` except it still includes certain symbols such as `$` while `W` will skip over `$`.
  
For example instead of moving from (carots represent where cursor should be moving):
```
"hello" + "world";
^^    ^ ^ ^^    ^^
```

I am now getting it moving from:
```
"hello" + "world";
^       ^ ^
```

As you can see, it also now avoids punctuation like semi-colons and will instead just move to the next line.

If I delete or change a word `ciw` `diw` it includes the first quote in deletion and does not delete inside of the quotes unless I specify it with `ci"`, `di"`, `ct"`, or `dt"`.  This happens for all quotes whether they be double quotes, single quotes, or backticks.  I'm assuming this is a bug that occurred in one of the latest updates.  Again, it seems as if it is now treating all `w`s as `W`s with some odd exceptions.  I have tried reinstalling the extension as well as commenting out all of my custom settings to no avail.  I have also tested this on both my work Windows laptop and my personal MacOS (M1 Max) laptop and they both have the same issue.

EDIT: I've tried installing previous versions of the extension and I'm still encountering the same issue.  I'm assuming this has to do with a VSCode update rather than the extension itself?  Or possibly a conflict with another extension?  Seems extremely odd.

EDIT2: I've tried older versions of VSCode and still haven't been able to fix the issue.  Also tried uninstalling recently installed extensions.  There is one situation where I edited the setting for `isKeyword`.  However, I uncommented that setting a while back as I found it wasn't working as I had expected it to.  Maybe it hasn't reverted back to its' default setting?

**Comment by w-cantin** [comment]
Hello,

The issue is probably related to your settings as I cannot reproduce the issue. If I am not mistaken the words separators considered by VSCodeVim are the native VSCode's ones when the command `vim.iskeyword` is not overwritten. Have you tried resetting VSCode's `Word Separators` setting?

**Comment by MatthewCushing** [comment]
I have not.  Let me look into that setting and hopefully I come back with positive results!

**Comment by MatthewCushing** [comment]
Wow!  I had no idea I had modified that setting when I was testing out the Tabnine extension quite a long time ago.  Commenting out that setting did the trick!  Thank you so much!  You have no idea how infuriating this has been for me.  Apologize for creating the issue here when it had nothing to do with you but I'm extremely happy I did.  Otherwise I would still be cursed with this.  Thanks again @w-cantin
